### PR TITLE
Show overdue search history when search query is empty

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.shortcut.com/simpledotorg/story/8520/display-search-history-in-overdue-search-screen

### CODE REVIEW

**Comment by msasikanth** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3951** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/3951" target="_blank">Graphite</a> 
    * **PR #3952** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/3952" target="_blank">Graphite</a> 
      * **PR #3953** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/3953" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/3953?utm_source=stack-comment).

**Review by janhavisinghh — APPROVED** [review]

**Comment by msasikanth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/3953?utm_source=stack-comment).


Job Reference: QO6BNrFvMbihWnEksjTV

**Review by janhavisinghh — APPROVED** [review]

**Comment by msasikanth** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/3953?utm_source=stack-comment).


Job Reference: MBc8XzaLzsFbKLEEof5X

**Pull request merged**
